src/biome/text/pipeline.py

```diff
@@ -154,6 +156,64 @@ def init_prediction_cache(self, max_size: int) -> None:
         """
         self._model.init_prediction_cache(max_size)
 
+    def find_lr(
+        self,
+        trainer_config: TrainerConfiguration,
+        find_lr_config: FindLRConfiguration,
+        training_data: Union[DataSource, InstancesDataset],
+    ):
+        """Returns a learning rate scan on the model.
+
+        It increases the learning rate step by step while recording the losses.
+
+        Parameters
+        ----------
+        trainer_config
+            A trainer configuration
+        find_lr_config
+            A configuration for finding the learning rate
+        training_data
+            The training data
+
+        Returns
+        -------
+        (learning_rates, losses)
+            Returns list of learning rates and corresponding losses.
+            Note: The losses are recorded before applying the corresponding learning rate
+        """
```

Comment:
It would be nice include a minimal code snippet of how to use this method for learning rate selection (if possible)

Reply:
Sorry, i totally forgot about this comment!
I planed to include a more detailed usage in one or two of our tutorials. There are a lot of blog posts/publications on how to select the learning rate once you made a learning rate scan, in my experience this learning rate scan gives you a rough estimate where to look for the learning rate, but to find the optimal one you still have to do a bit of HPO. I will include a link in the docstring pointing to one of those blog posts.